README.md

```diff
@@ -182,8 +182,19 @@ To use this component,
 )(contentBlock)
 ```
 
-If you additionally need the [HMRC banner](https://design.tax.service.gov.uk/hmrc-design-patterns/hmrc-banner/) –
-most services do not – set `displayHmrcBanner` to true.
+hmrcStandardHeader can also be configured to generate the GOV.UK phase banner, HMRC user research 
+recruitment banner and the [HMRC banner](https://design.tax.service.gov.uk/hmrc-design-patterns/hmrc-banner/)
+ inside the HEADER tag. For example,
+
+```
+hmrcStandardHeader(
+  serviceUrl = Some(controllers.routes.IndexController.index().url),
+  signOutUrl = Some(controllers.routes.SignOutController.signOut().url),
+  phaseBanner = Some(StandardPhaseBanner(phase = "alpha", url = appConfig.feedbackUrl)),
```

Comment:
This still shows the old API, not `Some(StandardAlphaBanner(url = appConfig.feedbackUrl))`